ts-types/value/ensure.d.ts

```diff
@@ -0,0 +1,4 @@
+import { EnsureOptions } from '../ensure';
+
+declare function ensureValue(value: any, options?: EnsureOptions): any;
```

Comment:
Here I believe we can also do `enureValue<T>..` as it's always input that'll be returned